### PR TITLE
Jira DOC-894: Add aliases for "unbranched" requests

### PR DESCRIPTION
**Pull request by lanceleonard**

### CODE REVIEW

**Comment by lanceleonard** [comment]
A staged preview of this work [is available](https://docs.redis.com/staging/jira-doc-894/rc/).

File: content/modules/redisbloom/_index.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ weight: 30
 alwaysopen: false
 categories: ["Modules"]
 aliases:
+  - /redisbloom/
+  - /redis-bloom/
+  - /-redis_bloom/
```
Comment (rrelledge) [inline content/modules/redisbloom/_index.md:10]:
```suggestion
  - /redis_bloom/
```

File: content/modules/redistimeseries/_index.md [hunk 2]
```diff
@@ -5,6 +5,9 @@ weight: 80
 alwaysopen: false
 categories: ["Modules"]
 aliases:
+  - /redistimeseries/
+  - /redis-time-series/
+  - /redis/_time_series/
```
Comment (rrelledge) [inline content/modules/redistimeseries/_index.md:10]:
```suggestion
  - /redis_time_series/ 
```

**Review by kaitlynmichael — APPROVED** [review]

File: website.json [hunk 3]
```diff
@@ -66,6 +66,156 @@
...
+            },
+            "Redirect": {
+                "HostName": "docs.redis.com",
+                "Protocol": "https",
+                "ReplaceKeyPrefixWith": "latest/modules/"
+            }
+        },
+        {
+            "Condition": {
+                "KeyPrefixEquals": "platforms"
```
Comment (rrelledge) [inline website.json:111]:
```suggestion
                "KeyPrefixEquals": "platforms/"
```

File: website.json [hunk 4]
```diff
@@ -66,6 +66,156 @@
...
+        {
+            "Condition": {
+                "KeyPrefixEquals": "redistimeseries/"
+            },
+            "Redirect": {
+                "HostName": "docs.redis.com",
+                "Protocol": "https",
+                "ReplaceKeyPrefixWith": "latest/modules/redistimeseries"
+            }
+        },
```
Comment (rrelledge) [inline website.json:218]:
```suggestion
        },
        {
            "Condition": {
                "KeyPrefixEquals": "glossary/"
            },
            "Redirect": {
                "HostName": "docs.redis.com",
                "Protocol": "https",
                "ReplaceKeyPrefixWith": "latest/glossary"
            }
        },
```

**Review by rrelledge — APPROVED** [review]
I think this looks good. I just left a few typo fix suggestions and also added a suggestion for adding a glossary redirect in website.json.

**Pull request merged**